src/bodywork/cli/cli.py

```diff
@@ -548,23 +530,6 @@ def stage(args: Namespace) -> None:
         sys.exit(1)
 
 
-@handle_k8s_exceptions
-def workflow(args: Namespace) -> None:
-    """Workflow execution handler
-
-    :param args: Arguments passed to the workflow command from the CLI.
-    """
-    try:
-        repo_url = args.git_url
-        repo_branch = args.git_branch
-        docker_image = args.bodywork_docker_image
-        load_kubernetes_config()
-        run_workflow(repo_url, repo_branch, docker_image_override=docker_image)
-        sys.exit(0)
-    except BodyworkWorkflowExecutionError:
```

Comment:
I think we need to retain this block.

Reply:
Same as your comment above about the Try and Except block, no?

Reply:
Yes - all a bit stream-of-consciousness at the moment...